playbooks/openshift-master/private/config.yml

```diff
@@ -192,6 +206,17 @@
     openshift_master_default_registry_value: "{{ hostvars[groups.oo_first_master.0].l_default_registry_value }}"
     openshift_master_default_registry_value_api: "{{ hostvars[groups.oo_first_master.0].l_default_registry_value_api }}"
     openshift_master_default_registry_value_controllers: "{{ hostvars[groups.oo_first_master.0].l_default_registry_value_controllers }}"
+    when: not ( openshift_master_bootstrap_enabled | default(false) | bool )
+
+  - role: openshift_control_plane
+    openshift_master_hosts: "{{ groups.oo_masters_to_config }}"
```

Comment:
'openshift_master_hosts' does not appear to be used in openshift_control_plane role.